test/testmachinery/shoots/operations/containerruntime.go

```diff
@@ -95,10 +95,6 @@ var _ = Describe("Shoot container runtime testing", func() {
 		containerdServiceCommand := []string{"systemctl", "is-active", "containerd"}
 		executeCommand(ctx, f, rootPodExecutor, containerdServiceCommand, "active")
 
-		// check that config.toml is configured
-		checkConfigurationCommand := []string{"sh", "-c", "cat /etc/systemd/system/containerd.service.d/11-exec_config.conf | grep 'usr/bin/containerd --config=/etc/containerd/config.toml' | echo $?"}
```

Comment:
What do we do about other OS extensions which still create this file? Is it fine to neglect them or will they also be cleaned up? Any thoughts @Roncossek or @MrBatschner?